jetson/percep/main.cpp

```diff
@@ -155,10 +148,13 @@ int main() {
   while (true) {
```

Comment:
The j has been there since before I was on the team. It is used for image writing, so that the program can run for infinite time, but keep writing disctinct images. We'll work on that

Reply:
#435

Reply:
I have done so